README.md

```diff
@@ -12,17 +13,23 @@
 
 ## ⚒️ Built With
 
-- 
-- 
-
+- BOOTSTRAP
+- CSS
+- JAVASCRIPT
+- GRID
+- FLEX
+- LINTERS (stylint and eslint)
 ## Live Demo
-- [Coming soon]
+- [g-summit](https://altontonn.github.io/Global-Summit/)
+- [presentation](https://drive.google.com/file/d/1XeBZOdulTpzbyr3YAuACGjbL_1NjGHw1/view?usp=sharing)
```

Comment:
- [ ] Kindly ensure to describe what your project is about, your video only highlighted the features, also add a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
Hey, should i show the code during my presentation